src/Controllers/CourseUserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Uasoft\Badaso\Module\LMSModule\Controllers;
+
+use Exception;
+use Illuminate\Validation\ValidationException;
+use Uasoft\Badaso\Controllers\Controller;
+use Uasoft\Badaso\Helpers\ApiResponse;
+use Uasoft\Badaso\Module\LMSModule\Models\Course;
+use Uasoft\Badaso\Module\LMSModule\Models\CourseUser;
+use Uasoft\Badaso\Module\LMSModule\Models\User;
+
+class CourseUserController extends Controller
+{
+    public function view()
+    {
+        try {
+            $user = auth()->user();
+
+            $courseUsers = CourseUser::where(
+                'user_id', '=', $user->id
+            )->pluck('course_id')->toArray();
+            $courses = [];
+
+            foreach ($courseUsers as $cid) {
```

Comment:
then this becomes `as $courseUserId`, I prefer write it in a verbose way, so people in the future won't have to "guess" the content of the variable